rotkehlchen/exchanges/binance.py

```diff
@@ -589,6 +607,140 @@ def _query_lending_balances(
 
         return balances
 
+    def _query_lending_interests_history(
+            self,
+            cursor: 'DBCursor',
+            start_ts: Timestamp,
+            end_ts: Timestamp,
+    ) -> None:
+        """Queries Binance lending interest history, transforms it
+        into `HistoryBaseEntry` objects and save it in the database.
+        May raise:
+        - RemoteError
+        """
+        ranges = DBQueryRanges(self.db)
+        range_query_name = f'{self.location}_history_events_{self.name}'
+        ranges_to_query = ranges.get_location_query_ranges(
+            cursor=cursor,
+            location_string=range_query_name,
+            start_ts=start_ts,
+            end_ts=end_ts,
+        )
+        for query_start_ts, query_end_ts in ranges_to_query:
+            events = []
+            for lending_type in ('DAILY', 'ACTIVITY', 'CUSTOMIZED_FIXED'):
```

Comment:
what is the difference between this 3 types?

Reply:
It represents the various lending types supported.